### PR TITLE
Issue/5 project restructure

### LINKED ISSUE #5: Project restructure to make it a library
The structure of the project should follow along the lines of https://github.com/wordpress-mobile/react-native-recyclerview-list

### PR DESCRIPTION
**Pull request by mzorz**
Fixes #5 

This PR brings the new library structure as defined and proposed by @maxme following the https://github.com/wordpress-mobile/react-native-recyclerview-list project

```
.
├── example
│   ├── android
│   ├── ios (doesn't exist yet)
│   ├── app.json
│   ├── index.android.js
│   ├── index.ios.js (doesn't exist yet)
│   └── package.json
├── android
│   └── src
│      └── main
│          └── java
│              └── org
│                  └── wordpress
│                      └── mobile
│                          └── ReactNativeAztec
│                              ├── ReactAztecBlurEvent.java
│                              ├── ReactAztecEndEditingEvent.java
│                              ├── ReactAztecFocusEvent.java
│                              ├── ReactAztecManager.java
│                              ├── ReactAztecPackage.java
│                              ├── ReactAztecText.java
│                              └── ReactAztecView.java
├── src
│   └── AztecView.js
├── index.js
├── package.json
├── LICENSE.md
└── README.md
```

Thank you @maxme for the guidance and plumbing work

cc @daniloercoli @maxme

### CODE REVIEW

File: example/android/app/src/main/java/com/example/android/MyFragment.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         ReactRootView reactRootView = new ReactRootView(getContext());
         Bundle RNProps = new Bundle();
         RNProps.putString("text", EXAMPLE);
```
Comment (maxme) [inline example/android/app/src/main/java/com/example/android/MyFragment.java]: I'm not sure why we have this on the Java side. Should we set the text from the js example file instead?

Comment (daniloercoli) [reply example/android/app/src/main/java/com/example/android/MyFragment.java]: I agree, we should set the content from JS at this point of the project.

Comment (mzorz) [reply example/android/app/src/main/java/com/example/android/MyFragment.java]: sounds good! addressed in 0dc4dd4

**Comment by daniloercoli** [comment]
Looks good! :shipit:

**Pull request merged**